tests/aws/templates/macros/return_random_string.py

```diff
@@ -0,0 +1,14 @@
+import random
+import string
+
+
+def handler(event, context):
+    parameters = event["templateParameterValues"]
+    fragment = f"{parameters['Input']}-{random_string(5)}"
+    resp = {"requestId": event["requestId"], "status": "success", "fragment": fragment}
+
+    return resp
+
+
+def random_string(length):
+    return "".join(random.choice(string.ascii_lowercase) for i in range(length))
```

Comment:
nit: using a random string makes testing a bit unnecessarily hard, e.g. for adding snapshots. not a big issue though

Reply:
needed a simple idea for a macro. But I will keep it for mind next time.